model/pack/QtiItemPacker.php

```diff
@@ -127,12 +129,40 @@ public function packQtiItem($item, $lang, $qtiItem, Directory $directory)
         return $itemPack;
     }
 
+    /**
+     * @param $item
+     * @param $qtiItem
+     * @param Directory $directory
+     * @param PackedAsset[] $packedAssets
+     * @return ItemPack
+     * @throws common_Exception
+     */
+    public function createQtiItemPackWithAssets($item, $qtiItem, Directory $directory, array $packedAssets): ItemPack
```

Comment:
Second parameter is not necessary as it is not used anymore: https://github.com/oat-sa/extension-tao-itemqti/pull/1481/files#r450060355

Reply:
good catch